make.ps1

```diff
@@ -122,13 +122,20 @@ function Check-Command
 	{
 		echo "Checking for explicit interface violations..."
 		Invoke-Expression "$utilityPath $modID --check-explicit-interfaces"
+	}
+	else
+	{
+		UtilityNotFound
+	}
 
+	if (Test-Path "$env:ENGINE_DIRECTORY/OpenRA.StyleCheck.exe")
+	{
 		echo "Checking for code style violations in OpenRA.Mods.$modID..."
-		Invoke-Expression "$utilityPath $modID --check-code-style OpenRA.Mods.$modID"
+		Invoke-Expression "$env:ENGINE_DIRECTORY/OpenRA.StyleCheck.exe OpenRA.Mods.$modID"
 	}
 	else
 	{
-		UtilityNotFound
+		echo "OpenRA.StyleCheck.exe could not be found. Build the project first using the `"all`" command."
```

Comment:
It wouldn't hurt to print out the actual path here.

`echo "$env:ENGINE_DIRECTORY/OpenRA.StyleCheck.exe ..."